src/panel/spoiler_card/SpoilerCardList.js

```diff
@@ -11,7 +11,7 @@ const SpoilerCardList = observer(props =>
 				<SpoilerCardContainer
 					index={index}
 					title={obj["title"]}
-					tags={obj["tags"].join(", ")}
```

Comment:
What if we pretty print the tagString which they have given to us. Something like remove spaces after `(`, before `)` and ensure there are spaces around all the binary operators. It would take some doing though :weary: